### PR TITLE
docs(tutorial): finish step 1, other steps in draft

### PR DESCRIPTION
**Pull request by mattrosno**
Tutorial fixes for step 1 and 2 after getting design review, grid review, and two days of dry-run feedback.

### CODE REVIEW

File: src/content/tutorial/react-step-1/react-step-1.mdx [hunk 1]
```diff
@@ -131,7 +141,9 @@ In the `src` directory, rename `index.css` as `index.scss`. Then in `index.js` u
 
 ### Import carbon-component styles
 
-In `index.scss` import Carbon styles by adding:
+In `index.scss`, import the Carbon styles by adding the following at the top of the file:
+
+##### src/index.scss
 
 ```scss
 @import 'carbon-components/scss/globals/scss/styles.scss';
```
Comment (joshblack) [inline src/content/tutorial/react-step-1/react-step-1.mdx]: Should this be using `@import '~carbon-components'` or does it not matter?

File: src/content/tutorial/react-step-1/react-step-1.mdx [hunk 2]
```diff
@@ -141,22 +153,26 @@ This will take a moment for the Sass to compile. Once compiled, you'll notice th
...
 Because any change to `index.scss` will re-compile all of the Carbon Sass, create an `app.scss` file in the `src` directory and in `App.js`, import that new file.
 
+##### src/App.js
+
 ```javascript
 import './app.scss';
 ```
 
-_Note: To optimize our Sass compile time, we'll be adding app-specific styling to `app.scss` and only modifying `index.scss` when necessary._
+_Note: To optimize our Sass compile time, we'll be adding app-specific styling to_ `app.scss` _and only modifying_ `index.scss` _when necessary._
```
Comment (joshblack) [inline src/content/tutorial/react-step-1/react-step-1.mdx]: Does `app.scss` exist yet?

File: src/content/tutorial/react-step-1/react-step-1.mdx [hunk 3]
```diff
@@ -324,9 +350,7 @@ You should now see a styled UI Shell header and a button below it.
 
 ## Create pages
 
-Next thing we need to do is create the files for our content.
-
-Start by creating a folder called `content` in `src/`. This should be a sibling of `src/components`.
+Next thing we need to do is create the files for our content. Start by creating a folder called `content` in `src`. This should be a sibling of `src/components`.
```
Comment (joshblack) [inline src/content/tutorial/react-step-1/react-step-1.mdx]: And reason why we're using a `src/content` hierarchy? Another idea could be:

```
src/routes/index.js             <-- Capture all routes
src/routes/<route>/index.js     <-- Route entrypoint
src/routes/<route>/components   <-- Components specific to route
src/routes/<route>/routes/...   <-- Nested structure   
```

Where each route boundary is guarded with an `import()` to support route-based code splitting

File: src/content/tutorial/react-step-1/react-step-1.mdx (hunk 1 above)
Comment (mattrosno) [reply src/content/tutorial/react-step-1/react-step-1.mdx]: I haven't noticed it necessary. Maybe `~` was needed with an older version of `sass-loader`?

File: src/content/tutorial/react-step-1/react-step-1.mdx (hunk 2 above)
Comment (mattrosno) [reply src/content/tutorial/react-step-1/react-step-1.mdx]: Done in paragraph above.

File: src/content/tutorial/react-step-1/react-step-1.mdx (hunk 3 above)
Comment (mattrosno) [reply src/content/tutorial/react-step-1/react-step-1.mdx]: Going to hold off on this until step 5 when we're going through performance/build considerations, e.g. bundling.

**Pull request merged**